migtests/tests/pg/omnibus/expected_files/expected_schema_analysis_report.json

```diff
@@ -228,7 +228,7 @@
             "IssueType": "unsupported_features",
             "ObjectType": "INDEX",
             "ObjectName": "idx_1 ON composite_type_examples.ordinary_table",
-            "Reason": "INDEX on column 'user_defined_type' not yet supported",
+            "Reason": "Indexes on columns with complex data types like user_defined_type are not yet supported in YugabyteDB",
```

Comment:
nit: quote typename `user_defined_type` ?

Reply:
yeah that can be done.

Reply:
done